### PR TITLE
Add link to example tests

### PR DESCRIPTION
**Pull request by PavelPolyakov**
Link an example of the bot covered with tests

### CODE REVIEW

**Review by roj1512 — COMMENTED** [review]
I prefer putting a link to the test file in your repository, put it in the “Contribute a Testing Framework”, and talk about it as if it is an example how tests could be done. I’d also like to know @KnorpelSenf’s opinion.

**Comment by KnorpelSenf** [comment]
I tend to agree, but I'll leave the choice up to you guys.

**Comment by PavelPolyakov** [comment]
I moved it to the tip section. Though I linked the whole repository, this gives an option to update repository without updating this documentation. I think it's fine, since that repo has proper `README.md`.

File: site/docs/advanced/deployment.md [hunk 1]
```diff
@@ -61,4 +61,6 @@ This can be done with grammY like so:
 While grammY provides the necessary hooks to start writing tests, it would be very helpful to have a testing framework for bots.
 This is novel territory, such testing frameworks largely do not exist.
 We look forward to your contributions!
+
+[Example](https://github.com/PavelPolyakov/grammy-with-tests), which shows how tests could be done.
```
Comment (roj1512) [inline site/docs/advanced/deployment.md:65]:
```suggestion
An example on how tests could be done [can be found here](https://github.com/PavelPolyakov/grammy-with-tests/blob/master/__tests__/bot.test.js).
```

**Review by roj1512 — CHANGES_REQUESTED** [review]

**Review by roj1512 — APPROVED** [review]

**Review by WingLim — APPROVED** [review]
Done

**Pull request merged**